webrtc/RTCPeerConnection-tests.ts

```diff
@@ -1,6 +1,8 @@
 /// <reference path="MediaStream.d.ts" />
 /// <reference path="RTCPeerConnection.d.ts" />
 
+let voidpromise: Promise<void>;
```

Comment:
I don't think this abbreviation helps much; it's not really much more concise, and means that is someone jumps into the middle of the file, they have a to search and find this definition. I'd recommend just using the `Promise<void>` below instead.